.werft/build.js

```diff
@@ -62,6 +62,26 @@ async function build(context, version) {
         "HTTP_PROXY": "http://dev-http-cache:3129",
         "HTTPS_PROXY": "http://dev-http-cache:3129",
     };
+      var terraformScripts = [
+      "aws-full-terraform",
+      "aws-terraform",
+      "gcp-full-terraform",
+      "gcp-terraform"
+    ]
+
+    for await (var i of terraformScripts) {
+      try {
+        werft.log(`install/${i}`, "\x1b[1;34mInitializing\x1b[m")
+        exec(`cd install/${i} && terraform init -backend=false`, { slice: `install/${i}`})
+        werft.log(`install/${i}`, "\x1b[1;34mCheck Style\x1b[m")
+        exec(`cd install/${i} && terraform fmt -recursive -check`, {slice: `install/${i}`})
+        werft.log(`install/${i}`, "\x1b[1;34mValidating\x1b[m")
+        exec(`cd install/${i} && terraform validate`, { slice: `install/${i}`})
+        werft.done(`install/${i}`)
+      } catch (err) {
+        werft.fail(`install/${i}`, err)
+      }
+    }
```

Comment:
can you move the new code into a method? something like `validateTerraform()`.

Reply:
Done!